crates/forge/src/lib.rs

```diff
@@ -462,35 +487,101 @@ fn run_with_fuzzing(
     };
     let mut fuzzer = RandomFuzzer::create(fuzzer_seed, fuzzer_runs, &args)?;
 
-    let mut results = vec![];
+    let mut tasks = FuturesUnordered::new();
+
+    let (send, mut recv) = channel(1);
```

Comment:
Do we use it anywhere?

Reply:
because we waiting for things to finish shutting down - https://tokio.rs/tokio/topics/shutdown